src/index.css

```diff
@@ -0,0 +1,379 @@
+@font-face {
+  font-family: 'Bitter';
+  src: local('Bitter'),
+    url(./fonts/Bitter-VariableFont_wght.ttf) format('truetype');
+}
+
+@font-face {
+  font-family: 'Montserrat';
+  src: local('Montserrat') url(./fonts/Montserrat-Regular.ttf)
+    format('truetype');
+  font-weight: 400;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Montserrat';
+  src: local('Montserrat') url(./fonts/Montserrat-Medium.ttf) format('truetype');
+  font-weight: 500;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Montserrat';
+  src: local('Montserrat') url(./fonts/Montserrat-Bold.ttf) format('truetype');
+  font-weight: 700;
+  font-style: normal;
+}
+
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
Your code looks solid. One hint though: The disadvantage of copying the normalize.css code into your repo is that you need to maintain it. Let's say new browser version need changes in this code. You probably wouldn't realize. If you'd use an npm package instead you'd see at some point that there's a new version and could simply update it